src/NuGet.Core/NuGet.Common/PathUtil/PathUtility.cs

```diff
@@ -69,5 +70,16 @@ private static string EnsureTrailingCharacter(string path, char trailingCharacte
 
             return path + trailingCharacter;
         }
+
+        private static bool IsMacOSCaseSensitive()
```

Comment:
why do this when we already know that windows will be always case-insensitive and linux will always be case sensitive?